public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -1,33 +1,41 @@
 window.addEventListener("load", createSidebar);
-window.addEventListener('load', rtdefault);
 window.addEventListener('load', TOC);
-//window.addEventListener("load", getCC);
-window.addEventListener("load", getattrText);
+window.addEventListener("load", getCC);
+
+
+window.addEventListener("load", () => {
+  console.log(localStorage.getItem('page_width'));
+  $('section.mt-content-container').css("margin-left", localStorage.getItem('page_width') + "px");
+  $('section.mt-content-container').css("margin-right", localStorage.getItem('page_width') + "px");
+  $("#slider-page-width").val(localStorage.getItem('page_width'));
+});
 
 function createSidebar() {
-	const isPro = document.getElementById("proHolder").innerText === 'true';
-	let upperDiv = document.createElement("div");
-	
-	document.body.append(upperDiv);
-	
-	if (isPro) {
-		upperDiv.innerHTML = `
+  const isPro = document.getElementById("proHolder").innerText === 'true';
+  let upperDiv = document.createElement("div");
+
+  document.body.append(upperDiv);
+
+  if (isPro) {
+    upperDiv.innerHTML = `
```

Comment:
This does not need to be solved now, but are there parts of this if/else statement that are identical and thus can be reused between the two?